build-packages.sh

```diff
@@ -110,20 +110,20 @@ export DEBEMAIL="Confluent Packaging <packages@confluent.io>"
 xmlstarlet ed --inplace -P --update "/_:project/_:parent/_:version" --value "${UPSTREAM_VERSION}" pom.xml
 
 # Maven provides some helpfull commands for setting versions/properties
-mvn --batch-mode versions:set          -DgenerateBackupPoms=false "-DnewVersion=${FULL_VERSION}"
-mvn --batch-mode versions:set-property -DgenerateBackupPoms=false "-DnewVersion=${FULL_VERSION}" -Dproperty=io.confluent.ksql.version 
+mvn --batch-mode versions:set          -DgenerateBackupPoms=false "-DnewVersion=${VERSION}"
+mvn --batch-mode versions:set-property -DgenerateBackupPoms=false "-DnewVersion=${VERSION}" -Dproperty=io.confluent.ksql.version
 mvn --batch-mode versions:set-property -DgenerateBackupPoms=false "-DnewVersion=${UPSTREAM_VERSION}" -Dproperty=io.confluent.schema-registry.version
 
 # Set version for Debian Package
-dch --newversion "${FULL_VERSION}" "Release version ${FULL_VERSION}"
+dch --newversion "${VERSION}" "Release version ${VERSION}"
```

Comment:
```suggestion
dch --newversion "${FULL_VERSION}" "Release version ${FULL_VERSION}"
```

I'm seeing this error in the PR build:

```
14:19:13  gbp:info: confluent-ksqldb_None.orig.tar.gz does not exist, creating from 'debian-0.18.0'
```

Which causes issues later on, in my local docker experiments changing this back to FULL_VERSION gets it working.